6502.js

```diff
@@ -596,7 +599,25 @@ export function Cpu6502(model, dbgr, video_, soundChip_, ddNoise_, cmos, config)
             return this.ramRomOs[this.osOffset + (addr & 0x3fff)];
         }
         addr &= 0xffff;
+            
+        if (model.hasMusic5000)
```

Comment:
For all these `model.has*` checks it might be simpler and less error-prone to check `if (this.music5000)` instead . That way whether the model said it had a 5000 or not, we don't try and call methods on an object that might not be there. (ditto teletext)

Reply:
...for example if you pick a music5000 model BUT the audio worklet fails to initialise (as is happening on my machine) you don't get a crash here and in other place)

Reply:
changed to `this.music5000` etc...

Reply:
You :star: - thanks!